amperity_reference/source/workflows.rst

```diff
@@ -702,13 +702,45 @@ Databases
 
 Some common workflow actions that your brand may see when generating databases include:
 
+* :ref:`Cannot bind inputs to resolved state <workflows-databases-inputs-unbound>`
 * Errors resolving column names in tables
 * Missing tables
 * SQL query alerts (including warning *and* failure thresholds)
 
 .. workflows-context-databases-end
 
 
+.. _workflows-databases-inputs-unbound:
+
+Cannot bind inputs to resolved state
+++++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. workflows-databases-inputs-unbound-start
+
+Inputs to the database build process, such as source and custom domain tables, stitched tables, and core tables, must have a schema that matches the one that is expected by the database.
+
+If there is a schema mismatch you will see an error similar to:
+
+::
+
+   Failure binding configured inputs to resolved state
+
+that is followed by a list of tables, and then for each table a list of fields that are causing the mismatch.
+
+In many cases this error is caused because the database run is occuring too soon after the schema change and the expected state has not been refreshed to match the schema change. This means that in many cases, waiting a few minutes, and then re-running the workflow is all that is necessary to resolve the workflow error.
```

Comment:
I might reword this a bit to more precisely match reality:

```suggestion
A common cause of this error is attempting to run databases immediately after making a schema change to a domain table, and before the corresponding schema migration workflow has completed. This can also occur after a sandbox merge if the merge included any schema changes. In this case the schema mismatch error can often be resolved by simply waiting a few minutes for any pending schema migrations to finish, and then re-running the database.
```

We can iterate on the exact phrasing.